File: Web-SPA/src/app/entities/address.ts
export class Address {
    city: string;
    state: string;
    lon: number;
    lat: number;

    constructor(city?: string, state?: string, lon?: number, lat?: number) {
        this.city = city || '';
        this.state = state || '';
        this.lat = lat || 0;
        this.lon = lon || 0;
    }
}

File: Web2-Microservices/AirlineMicroservice/Models/Address.cs
using System;
using System.Collections.Generic;
using System.Linq;
using System.Threading.Tasks;

namespace AirlineMicroservice.Models
{
    public class Address
    {
        public Address()
        {

        }
        public int AddressId { get; set; }
        public string City { get; set; }
        public string State { get; set; }
        public double Lat { get; set; }
        public double Lon { get; set; }
        public int AirlineId { get; set; }
        public virtual Airline Airline { get; set; }
    }
}
